WordPress/src/main/res/values/attrs.xml

```diff
@@ -29,6 +29,8 @@
     <declare-styleable name="LearnMorePreference">
         <attr name="url" format="string" />
         <attr name="caption" format="string" />
+        <attr name="button" format="string" />
+        <attr name="learnMoreIcon" format="reference" />
```

Comment:
Is there a conflict with naming this attribute just `icon` rather than `learnMoreIcon`?  Using `icon` would be more consistent with the other `url`, `caption`, and `button` attributes.

Reply:
I had some build issues before with naming the attribute but now it seems to work.